<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="commitButton">
      <property name="text">
       <string>Commit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextBrowser" name="fromText"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="copyFromButton">
      <property name="text">
       <string>Copy from file</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="copyToButton">
      <property name="text">
       <string>Copy to file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="toLabel">
      <property name="text">
       <string>Selected file: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="fromLabel">
      <property name="text">
       <string>Selected file: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextBrowser" name="toText"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
